src/App/Pages/Vault/CipherSelectionPage.xaml.cs

```diff
@@ -51,10 +68,16 @@ protected async override void OnAppearing()
                 return;
             }
 
-            _accountAvatar?.OnAppearing();
-            _vm.AvatarImageSource = await GetAvatarImageSourceAsync();
+            // TODO: There's currently an issue on iOS where the toolbar item is not getting updated
+            // as the others somehow. Removing this so at least we get the circle with ".." instead
+            // of a white circle
+            if (Device.RuntimePlatform != Device.iOS)
```

Comment:
Is it possible the change of `Priority` of the avatar from `-1` to `-2` broke the original colorization trick you did for iOS?  (I don't see any hardcoded positions but I may be missing something)

Reply:
It's possible and I think I didn't try that but I've created another task to track this issue so that I make progress with the whole feature. On this PR I won't be fixing this given that's something minor and can be fixed later on.